include/dlr_tvm.h

```diff
@@ -3,6 +3,7 @@
 
 #include <graph/graph_runtime.h>
 #include <tvm/runtime/memory.h>
+#include <json.hpp>
```

Comment:
Documentation recommends to include it the following way `#include <nlohmann/json.hpp>` https://github.com/nlohmann/json#integration.

Reply:
BTW, `json.hpp` is included to `dlr_common.h` already.

Reply:
I don't think that is a mandate. It depends how we pull the file and what directory use in includes.

I can get rid of json.hpp here.